index.html

```diff
@@ -11,59 +11,266 @@
+                    <p class="text1">html</p>
+                    <p class="text2">css</p>
+                    <p class="text3">javascript</p>
+                </div>
+                <div class="action2">
+                    <div class="see2">
+                        <div class="see-it">
+                            <article> See Project</article>
+                        </div>
+                    </div>
+                </div>
+            </div>
 
         </section>
+        <section class="container4">
+            <div class="right-block2">
+
+            </div>
+            <div class="left-block">
+                <div class="frame0">
+                    <div class="title2">Tonic</div>
+                    <div class="sub">
+                        <p class="can">CANOPY</p>
+                        <i class="list11"> Back End Dev</i>
+                        <i class="list22"> 2015 </i>
+
+                    </div>
+                </div>
+                <div class="primary-text2">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</div>
+                <div class="tags2">
+                    <p class="text1">html</p>
+                    <p class="text2">css</p>
+                    <p class="text3">javascript</p>
+                </div>
+                <div class="action2">
+                    <div class="see2">
+                        <div class="see-it">
+                            <article> See Project</article>
+                        </div>
+                    </div>
+                </div>
 
-    </div>
+            </div>
+        </section>
+        <section class="container5">
+            <div class="right-block3">
+
+            </div>
+            <div class="left-block">
+                <div class="frame0">
+                    <div class="title2">Multi-Post Stories</div>
+                    <div class="sub">
+                        <p class="can">CANOPY</p>
+                        <i class="list11"> Back End Dev</i>
+                        <i class="list22"> 2015 </i>
+
+                    </div>
+                </div>
+                <div class="primary-text2">A daily selection of privately personalized reads; no accounts or
+                    sign-ups
+                    required.</div>
+                <div class="tags2">
+                    <p class="text1">html</p>
+                    <p class="text2">css</p>
+                    <p class="text3">javascript</p>
+                </div>
+                <div class="action2">
+                    <div class="see2">
+                        <div class="see-it">
+                            <article> See Project</article>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </section>
+        <section class="container6">
+            <div class="acco">
+                <h1 class="about"> About Myself</h1>
```

Comment:
- [ ] As requested by the previous reviewer, kindly entire that you use the correct font size for this section.

**This is how I am seeing this section right now:**

![image](https://user-images.githubusercontent.com/86318284/193064827-b98a015f-2fbc-4ece-b77d-a9dfc83b5007.png)

**This is how it is supposed to be like:**

![image](https://user-images.githubusercontent.com/86318284/193064946-dce7a74f-caf6-4d06-9d1b-6b705739bdf9.png)